test/cli/test_cli_mask.py

```diff
@@ -85,3 +85,65 @@ def test_cli_mask_threshold():
         assert result.exit_code == 0
         assert result is not None
         assert np.all(mask[58:62, 28:31, 7:9] == expected)
+
+
+def test_cli_mask_sct_default():
```

Comment:
A recommendation is to comment above each test with the specific scenereo that would lead to that circumstance you're testing against.  eg: "User used input type incorrectly as x and not y" , "Input x was missing" etc. , "Happy Path" (works as expected).